### ISSUE TITLE
Propose adding a layout page and use the Theme wrapper at that level.

### ISSUE THREAD

**Issue by pkd2512** [reuters-graphics/bluprint_graphics-kit#105]
Maybe even move the analytics and headers to that level, so `page.svelte` looks cleaner. Can have some smart checks for embeds to not include the header maybe.

**Comment by hobbes7878** [comment]
I don't think we can do this because the outer `+layout` will override embeds, which should _technically_ always be on white background. (I know we tend not to reverse it for a full page embed, but still want the split between pages and embeds baked in, not something users have to override to do correctly.)
